src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java

```diff
@@ -111,6 +130,54 @@ void initialBeforeYouStart_NoDocumentCollection_shouldReturnWithoutUrl() {
                 .isEqualTo(expected);
     }
 
+    @Test
+    void generateJurisdictionCodesHtml() {
+        CaseData caseData = new CaseData();
+        addJurCodeToExistingCollection(caseData, DAG);
+
+        assertEquals(String.format(JUR_CODE_HTML, String.format(CASE_NAME_AND_DESCRIPTION_HTML, DAG,
+                JurisdictionCode.valueOf(DAG).getDescription())),
+            et1VettingService.generateJurisdictionCodesHtml(caseData.getJurCodesCollection()));
```

Comment:
Prefer Hamcrest...
```
assertThat(...)
.isEqualTo(...);
```
It makes for easier reading.
Please also apply to the rest of this PR too ;)